Packs/AzureFirewall/Integrations/AzureFirewall/AzureFirewall.yml

```diff
@@ -0,0 +1,1507 @@
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: The page number of the results to retrieve. Minimum value is 1.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    deprecated: false
+    description: List IP groups in resource group or subscription.
+    execution: false
+    name: azure-firewall-ip-group-list
+    outputs:
+    - contextPath: AzureFirewall.IPGroup.id
+      description: IP group resource ID.
+      type: String
+    - contextPath: AzureFirewall.IPGroup.name
+      description: IP group resource name.
+      type: String
+    - contextPath: AzureFirewall.IPGroup.properties.ipAddresses
+      description: List of IP addresses or IP address prefixes in the IP groups resource.
+      type: String
+  - arguments:
+    - default: false
+      description: Comma-separated list of IP group names resources to retrieve.
+      isArray: false
+      name: ip_group_names
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'True'
+      description: 'Indicates if the command was scheduled. Possible values: "True"
+        or "False".'
+      isArray: false
+      name: polling
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '30'
+      description: Indicates how long to wait between command execution (in seconds)
+        when 'polling' argument is true. Minimum value is 10 seconds.
+      isArray: false
+      name: interval
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '60'
+      description: Indicates the time in seconds until the polling sequence timeouts.
+      isArray: false
+      name: timeout
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieve IP group information.
+    execution: false
+    polling: true
+    name: azure-firewall-ip-group-get
+    outputs:
+    - contextPath: AzureFirewall.IPGroup.id
+      description: IP group resource ID.
+      type: String
+    - contextPath: AzureFirewall.IPGroup.name
+      description: IP group resource name.
+      type: String
+    - contextPath: AzureFirewall.IPGroup.properties.ipAddresses
+      description: List of IP addresses or IP address prefixes in the IP groups resource.
+      type: String
+  - arguments:
+    - default: false
+      description: Comma-separated list of IP group names resource to delete.
+      isArray: true
+      name: ip_group_names
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete IP group resource.
+    execution: false
+    name: azure-firewall-ip-group-delete
+  dockerimage: demisto/crypto:1.0.0.27243
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+fromversion: 5.5.0
```

Comment:
```suggestion
fromversion: 6.0.0
```

Reply:
Updated to 6.2.0 since most of the commands are `scheduled` commands